torch/csrc/distributed/rpc/rref_impl.cpp

```diff
@@ -258,6 +259,28 @@ void OwnerRRef::setError(std::exception_ptr eptr) {
   future_->setErrorIfNeeded(std::move(eptr));
 }
 
+void OwnerRRef::recordAllStreams(std::shared_ptr<LazyStreamContext> ctx) {
```

Comment:
looks like this can be `const std::shared_ptr<LazyStreamContext>&` instead?

Reply:
addressed